apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -39,60 +26,88 @@ type Story = StoryObj<typeof Tooltip>;
 
 export const TopPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
```

Comment:
I changed the way the `trigger` and `children` args are being defined on all these stories.

The problem is that passing them the way they were defined before doesn't generate the best code examples / UX in Storybook.

A better solution for a case like what we have here is to use the `render` function strategy as I've added to each story.